### DISCUSSION TITLE
Data transfer issues with Android's handwriting tablet

### DISCUSSION THREAD

**Discussion by xihuaxixuegui** [beeware/briefcase#1677] [Q&A]
I want to develop an Android drawing tablet through TOGA, is there any way to realize the sharing of hand-drawn drawings between the two drawing tablets; For example, I have two Android tablets with my app installed, numbered A and B; I want to realize the real-time display of the diagram drawn on tablet A on tablet B, is there any way to achieve this function?

**Comment by freakboy3742** [comment]
What you're describing sounds *possible* - I'd use a `toga.Canvas`, convert touch events on that canvas into drawing instructions to draw on the local Canvas instance, but also record all the drawing instructions. I'd then establish a communication link with the other tablet, communicate those drawing instructions to the other tablet, and reproduce the drawing instructions on the second tablet.

**Reply by xihuaxixuegui** [reply]
Is it possible for TOGA to implement this function at present?

**Reply by freakboy3742** [reply]
Yes, it should be possible. 

That doesn't mean it's going to be *simple*, but the complexity will be with the communications, not the parts that are Toga related.
